docs/workflows.rst

```diff
@@ -95,7 +95,7 @@ EPIMNITransformation
     :scale: 100%
 
 The EPIMNITransformation sub-workflow uses the transform from
-`ref_epi_t1_registration`_ and a T1-to-MNI transform from `t1w_preprocessing`_ to
+`EPIMeanNormalization`_ and a T1-to-MNI transform from `t1w_preprocessing`_ to
```

Comment:
Let's carefully inspect the docs that will get built on Circle. I expect this line to be problematic since `EPIMeanNormalization` does not exist.